src/Reflection/ReflectionObject.php

```diff
@@ -0,0 +1,465 @@
+<?php
+
+namespace BetterReflection\Reflection;
+
+use BetterReflection\Reflector\ClassReflector;
+use BetterReflection\SourceLocator\EvaledCodeSourceLocator;
+use PhpParser\Node\Stmt\Property as PropertyNode;
+use PhpParser\Builder\Property as PropertyNodeBuilder;
+
+class ReflectionObject extends ReflectionClass
```

Comment:
This is already done in [ReflectionObjectTest.php#126](https://github.com/Roave/BetterReflection/pull/92/files#diff-d4ef1e7f3bf562f17cbf82b1a40fa729R126)